src/test/java/ai/rev/speechtotext/integration/CustomVocabulariesTest.java

```diff
@@ -36,67 +39,77 @@
 
   @Test
   public void getCustomVocabularyInformation_IdIsSpecified_CustomVocabularyInformation() {
-    CustomVocabularyInformation submittedVocabulary = submitCustomVocabulary();
-    CustomVocabularyInformation retrievedVocabulary = null;
+    CustomVocabularySubmission customVocabularySubmission = createCustomVocabularySubmission();
+    CustomVocabularyInformation submittedVocabulary =
+        submitCustomVocabulary(customVocabularySubmission);
+    CustomVocabularyInformation retrievedVocabulary;
 
     try {
       retrievedVocabulary =
           customVocabularyClient.getCustomVocabularyInformation(submittedVocabulary.getId());
     } catch (IOException e) {
-      e.printStackTrace();
+      throw new RuntimeException(
+          "Error occurred getting the custom of vocabulary: " + e.getMessage());
     }
 
     assertThat(retrievedVocabulary.getMetadata())
         .as("Vocabulary metadata")
         .isEqualTo(submittedVocabulary.getMetadata());
-
     assertThat(retrievedVocabulary.getId())
         .as("Vocabulary Id")
         .isEqualTo(submittedVocabulary.getId());
-
     assertThat(retrievedVocabulary.getCreatedOn())
         .as("Created on date")
         .isEqualTo(submittedVocabulary.getCreatedOn());
-
+    assertThat(retrievedVocabulary.getCallbackUrl())
+        .as("Callback URL")
+        .isEqualTo(submittedVocabulary.getCallbackUrl());
     assertThat(retrievedVocabulary.getFailure()).as("Failure").isNull();
     assertThat(retrievedVocabulary.getFailureDetail()).as("Failure detail").isNull();
   }
 
   @Test
   public void getListOfCustomVocabulary_Called_ReturnListOfCustomVocabularyInformation() {
-    List<CustomVocabularyInformation> customVocabularies = null;
+    List<CustomVocabularyInformation> customVocabularies;
 
     try {
       customVocabularies = customVocabularyClient.getListOfCustomVocabularyInformation();
     } catch (IOException e) {
-      e.printStackTrace();
+      throw new RuntimeException(
+          "Error occurred getting the list of vocabularies: " + e.getMessage());
     }
 
     assertThat(customVocabularies.size()).as("List size").isGreaterThan(1);
     for (CustomVocabularyInformation customVocabulary : customVocabularies) {
+      CustomVocabularyStatus status = customVocabulary.getStatus();
       assertThat(customVocabulary.getId()).as("Vocabulary Id").isNotNull();
       assertThat(customVocabulary.getCreatedOn()).as("Created on").isNotNull();
-      assertThat(customVocabulary.getStatus()).as("Status").isNotNull();
-      if (customVocabulary.getStatus() == CustomVocabularyStatus.FAILED) {
+      assertThat(status).as("Status").isNotNull();
```

Comment:
I felt it was still needed because is should not be null and the next two if statements only check if it is failed or completed. This will ensure that it is always set to something.